CHANGELOG.md

```diff
@@ -1,8 +1,38 @@
 ## Next release
 
+## 0.39.0 / 2020-05-06
+
+* [CHANGE] Introduce release schedule (#3135)
+* [CHANGE] Remove Custom Resource Definition management and move CRD definitions to bundle.yaml (#3155, #3171)
+* [CHANGE] Remove apiextensions.k8s.io/v1beta1 CRD and require kubernetes v1.16 or newer (#3175, #3187)
```

Comment:
Instead of saying that we removed `apiextensions.k8s.io/v1beta1` CRD it might be better to say that we switched to `apiextensions.k8s.io/v1` CRD. What do you think?